src/index.html

```diff
@@ -20,8 +20,247 @@
+          </div>
+          <div class="card__review-text"><a>Reviews: 5</a></div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text"><a>Price:</a></div>
+          <div class="card__price-number"><a>$2,199</a></div>
+        </div>
+        <button class="card__button-buy">buy</button>
+      </div>
+
+      <div class="card">
+        <div class="card__logo"></div>
+        <div class="card__title">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+        <div class="card__code"><a>Product code: 195434</a></div>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text"><a>Reviews: 5</a></div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text"><a>Price:</a></div>
+          <div class="card__price-number"><a>$2,199</a></div>
+        </div>
+        <button class="card__button-buy">buy</button>
+      </div>
+
+      <div class="card">
+        <div class="card__logo"></div>
+        <div class="card__title">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+        <div class="card__code"><a>Product code: 195434</a></div>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text"><a>Reviews: 5</a></div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text"><a>Price:</a></div>
+          <div class="card__price-number"><a>$2,199</a></div>
+        </div>
+        <button class="card__button-buy">buy</button>
+      </div>
+
+      <div class="card">
+        <div class="card__logo"></div>
+        <div class="card__title">
+          <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+        </div>
+        <div class="card__code"><a>Product code: 195434</a></div>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__review-text"><a>Reviews: 5</a></div>
+        </div>
+        <div class="card__price">
+          <div class="card__price-text"><a>Price:</a></div>
+          <div class="card__price-number"><a>$2,199</a></div>
```

Comment:
```suggestion
          <a class="card__price-text">Price:</a>
          <a class="card__price-number">$2,199</a>
```
looks like you can simplify it